### PR TITLE
Create Testimonials Section

### PR DESCRIPTION
**Pull request by kulievtech**
Created Testimonials Section to show all the testimonials.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -284,5 +284,36 @@ <h4>Iswan Dracho</h4>
...
+                    <img src="./assets/customer-1.png" alt="James Steven">
+                    <h4>James Steven</h4>
+                    <h5>Uber Driver</h5>
+                    <p>
+                        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Soluta repudiandae cupiditate eaque obcaecati! 
+                    </p>
+                </div>
+                <div class="testimonials-section__testimonial">
+                    <img src="./assets/customer-3.png" alt="Tosh Mat">
+                    <h4>Tosh Mat</h4>
```
Comment (ohunazz) [inline index.html]: Wrong teacher name: change to Tosh Mattew

Comment (kulievtech) [reply index.html]: Good catch, I'll fix it.

Comment (ohunazz) [reply index.html]: lgtm

**Review by ohunazz — APPROVED** [review]
Good to go. You can merge to main.

**Pull request merged**